packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -112,3 +112,27 @@ void downloadFile(String src, String filename) {
   element.click();
   element.remove();
 }
+
+CoreElement createRecordingInstructions({@required String recordingGoal}) {
+  return div(c: 'center-in-parent recording-instruction-container')
+    ..layoutVertical()
+    ..flex()
+    ..add([
+      div(c: 'recording-instruction-message')
+        ..layoutHorizontal()
+        ..flex()
+        ..add([
+          div(text: 'Click the record button '),
+          createIconElement(record),
+          div(text: recordingGoal)
+        ]),
+      div(c: 'recording-instruction-message')
+        ..layoutHorizontal()
+        ..flex()
+        ..add([
+          div(text: 'Click the stop button '),
```

Comment:
Is there an intermediate step of: "trigger dart code to run that may have a performance problem" (although that message is too verbose so you would need something shorter).
I like that you included the icons in the message. That is slick.